lib/asyncapiSchemaFormatParser.js

```diff
@@ -43,15 +43,20 @@ async function parse({ message, originalAsyncAPIDocument, fileFormat, parsedAsyn
  * @private
  */
 function getMimeTypes() {
-  return [
-    ...['2.0.0', '2.1.0', '2.2.0', '2.3.0'].map((version) => {
-      return [
-        `application/vnd.aai.asyncapi;version=${version}`,
-        `application/vnd.aai.asyncapi+json;version=${version}`,
-        `application/vnd.aai.asyncapi+yaml;version=${version}`,
-      ];
-    }).flat().concat(['application/schema;version=draft-07', 'application/schema+json;version=draft-07', 'application/schema+yaml;version=draft-07'])
-  ];
+  const mimeTypes = [];
+  ['2.0.0', '2.1.0', '2.2.0', '2.3.0'].forEach(version => {
+    mimeTypes.push(...[
+      `application/vnd.aai.asyncapi;version=${version}`,
+      `application/vnd.aai.asyncapi+json;version=${version}`,
+      `application/vnd.aai.asyncapi+yaml;version=${version}`,
+    ]);
+  });
+  mimeTypes.push(...[
+    'application/schema;version=draft-07',
+    'application/schema+json;version=draft-07',
+    'application/schema+yaml;version=draft-07',
+  ]);
```

Comment:
You could simplify it a bit and do this: 

```suggestion
  const mimeTypes = ['application/schema;version=draft-07',
    'application/schema+json;version=draft-07',
    'application/schema+yaml;version=draft-07'];

  ['2.0.0', '2.1.0', '2.2.0', '2.3.0'].forEach(version => {
    mimeTypes.push(
      `application/vnd.aai.asyncapi;version=${version}`,
      `application/vnd.aai.asyncapi+json;version=${version}`,
      `application/vnd.aai.asyncapi+yaml;version=${version}`
    );
  });
```

Reply:
I wanna have `application/schema;version=draft-07` etc at the end of array as in original array :)

Reply:
but ok :)

Reply:
Ahh right 🙂 So this? 

```suggestion
  const mimeTypes = [];

  ['2.0.0', '2.1.0', '2.2.0', '2.3.0'].forEach(version => {
    mimeTypes.push(
      `application/vnd.aai.asyncapi;version=${version}`,
      `application/vnd.aai.asyncapi+json;version=${version}`,
      `application/vnd.aai.asyncapi+yaml;version=${version}`
    );
  });
  mimeTypes.push('application/schema;version=draft-07',
    'application/schema+json;version=draft-07',
    'application/schema+yaml;version=draft-07'
  );
```

Reply:
Ok, we can have that `application/schema;version=draft-07` at the beginning :)

Reply:
I propagated suggestions.